From dac5e0540fb0240a3319d3a3b655388d4351ce2b Mon Sep 17 00:00:00 2001
From: David Greaves <david.greaves@jolla.com>
Date: Wed, 18 Mar 2020 13:33:53 +0000
Subject: [PATCH 5/7] Use -v for --verbose and -V for --version

The old version of rpmlint used -v for verbose so this is better from a compatibility point-of-view

Additionally these are the traditional switches. See Eric Raymond's notes here:
http://catb.org/~esr/writings/taoup/html/ch10s05.html

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 rpmlint/__init__.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rpmlint/__init__.py b/rpmlint/__init__.py
index c094830..a295c61 100644
--- a/rpmlint/__init__.py
+++ b/rpmlint/__init__.py
@@ -39,7 +39,7 @@ def process_diff_args(argv):
                                      description='Shows basic differences between two rpm packages')
     parser.add_argument('old_package', metavar='RPM_ORIG', type=Path, help='the old package')
     parser.add_argument('new_package', metavar='RPM_NEW', type=Path, help='the new package')
-    parser.add_argument('-v', '--version', action='version', version=__version__, help='show package version and exit')
+    parser.add_argument('-V', '--version', action='version', version=__version__, help='show package version and exit')
     parser.add_argument('-i', '--ignore', nargs='+', default=None, choices=['S', 'M', '5', 'D', 'N', 'L', 'V', 'U', 'G', 'F', 'T'],
                         help="""file property to ignore when calculating differences.
                                 Valid values are: S (size), M (mode), 5 (checksum), D (device),
@@ -80,11 +80,11 @@ def process_lint_args(argv):
     parser = argparse.ArgumentParser(prog='rpmlint',
                                      description='Check for common problems in rpm packages')
     parser.add_argument('rpmfile', nargs='*', type=Path, help='files to be validated by rpmlint')
-    parser.add_argument('-v', '--version', action='version', version=__version__, help='show package version and exit')
+    parser.add_argument('-V', '--version', action='version', version=__version__, help='show package version and exit')
     parser.add_argument('-c', '--config', type=Path, help='load up additional configuration data from specified path')
     parser.add_argument('-e', '--explain', nargs='+', default='', help='provide detailed explanation for one specific message id')
     parser.add_argument('-r', '--rpmlintrc', type=Path, help='load up specified rpmlintrc file')
-    parser.add_argument('-V', '--verbose', '--info', action='store_true', help='provide detailed explanations where available')
+    parser.add_argument('-v', '--verbose', '--info', action='store_true', help='provide detailed explanations where available')
     parser.add_argument('-p', '--print-config', action='store_true', help='print the settings that are in effect when using the rpmlint')
     parser.add_argument('-i', '--installed', nargs='+', default='', help='installed packages to be validated by rpmlint')
     parser.add_argument('-s', '--strict', action='store_true', help='treat all messages as errors')
-- 
2.20.1

